android/package.json

```diff
@@ -15,13 +15,13 @@
 	],
 	"minSDKVersion": "16",
 	"vendorDependencies": {
-		"android sdk": ">=23 <=25.x",
-		"android build tools": ">=23 <=25.x",
-		"android platform tools": ">=17 <=25.x",
+		"android sdk": ">=25 <=26.x",
+		"android build tools": ">=25 <=26.x",
+		"android platform tools": ">=25 <=26.x",
 		"android tools": "<=26.x",
-		"android ndk": ">=r8e <=r9",
+		"android ndk": ">=r11c <=r15c",
```

Comment:
Yes, the min should be `r11c`, our `master` branch should be compatible with `r15c` with the new V8 update PR. So lets keep this in mind when we backport to `6_2_X`